### DISCUSSION TITLE
How upgrading php version

### DISCUSSION THREAD

**Discussion by HolgerAusB** [wallabag/wallabag#7888] [General]
A few weeks ago I wanted to check things for an other issue and changed the system standard php-version 8.2 to php8.3 in apache2's wallabag.conf:
```xml
<FilesMatch \.php$>
      SetHandler "proxy:unix:/run/php/php8.3-fpm.sock|fcgi://localhost"
</FilesMatch>
```
But I forgot to change this back and now I realized, that third party apps could not connect to the api or renew their token (Fiery Feeds and Frigolino), they are throwing OAuth2 errors. After switching back to php8.2 everything works well again.

So, how should I change the php-version, if I want to?

Personally I think, that I only need to install some additional [php8.3-extensions](https://doc.wallabag.org/en/admin/installation/requirements.html#php-extensions) but it could also be, that I need to do a another `make install` or `make update` or similar afterwards?

@nicosomb @j0k3r @yguedidi: Additionally I want to suggest:
- The `/about` page should name the php-version and maybe versions of other major components (composer, symphony...)
- Provide a php-file and/or a button on `/about` to 'check modules'. So we can see, which modules are already installed and which are missing.

**Comment by j0k3r** [comment]
The `install` command already check for required extensions.
And you're right, I think you don't just need to change the version, you also need to install all extensions required for that new PHP version.

Good point about PHP version, but maybe only for the admin user (just to avoid leaking information to anyone)
